nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -354,6 +354,11 @@
             <artifactId>nifi-database-utils</artifactId>
             <version>1.12.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>com.networknt</groupId>
+            <artifactId>json-schema-validator</artifactId>
+            <version>1.0.38</version>
```

Comment:
This dependency appears to be MIT licensed, which is acceptable in Apache terms, but it does need to be accounted for by updating the nifi-standard-nar's LICENSE file. This needs to also take into account any other transitive dependencies, which this brings in.

Reply:
The github repo states that the project is apache 2.0: [json-schema-validator/LICENSE](https://github.com/networknt/json-schema-validator/blob/master/LICENSE)
Did you look elsewhere to find a different license?